extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -38,8 +38,9 @@
 	'communitypage-user-rank' => 'Your Rank',
 	'communitypage-user-contributions' => '{{PLURAL:$1|contribution|contributions}}',
 	'communitypage-contributions' => '$1 {{PLURAL:$1|contribution|contributions}}',
-	'communitypage-admins' => 'Admins',
-	'communitypage-other-admins' => 'Other Admins',
+	'communitypage-admins' => 'Meet the Admins',
+	'communitypage-admins-welcome-text' => 'Ask us anything. We’d love to hear what brought you to {{SITENAME}}',
```

Comment:
I think wgWikiTopic won't be valid here. We want to hear what brought people to Wookiepedia, not what brought people to starwars :)